content/webapp/pages/exhibition-guide.tsx

```diff
@@ -122,7 +122,7 @@ const TypeOption: FC<TypeOptionProps> = ({
       >
         <h2 className="h2">{title}</h2>
         <p className={`${font('intr', 5)}`}>{text}</p>
-        {icon && <Icon icon={icon} />}
+        {icon ? <Icon icon={icon} /> : <Icon></Icon>}
```

Comment:
I think this should be `null` instead of `<Icon></Icon>`?

Reply:
or probably just

```
{icon && <Icon icon={icon} />}
```
?

Reply:
I was being ultra hacky, i want the space the icon tag creates, but obviously not the icon...this was my attempt to do that. I don't know if null gives me that.

Reply:
I don't think this will even build. I guess you could make the icon have a colour of `transparent`?

Reply:
```
import {indicator} from "the/path/to/icons";
// ...
{icon ? <Icon icon={icon} /> : <Icon icon={indicator} color="transparent"></Icon>}
```

`indicator` is just a solid circle in the icon set

Reply:
Is a Space component not an idea for this this because the icon size could change with different screen sizes?

Reply:
As I understand it, we want the amount of space to remain the same with and without a (visible) icon. I don't think our `Space` component is capable of this really (it's more for margin and padding), but rendering an _actual_ icon component without a visible colour should do it.

I guess an alternative would be to wrap the `Icon` in an element with a fixed height?

Reply:
I'll find less code-crime filled ways to sort this